proxy/reverse.go

```diff
@@ -98,6 +102,8 @@ func (p *reverseProxy) ServeHTTP(rw http.ResponseWriter, clientreq *http.Request
 			case <-closeNotifier.CloseNotify():
 				tp, ok := p.transport.(*http.Transport)
 				if ok {
+					atomic.StoreInt32(&requestCanceled, 1)
+					log.Printf("proxy: request from %v canceled", clientreq.RemoteAddr)
```

Comment:
do we really need this logging line? (we can add this back to debug level when we start to use leveled logging in proxy)